### PR TITLE
core/spell : fix Teleport With Error

### PR DESCRIPTION
**Pull request by cTrinity**
item:64457 The Last Relic of Argus  spell:82674 Teleport With Error

### CODE REVIEW

**Comment by Ovahlord** [comment]
where did you take the teleported coordinates from?

File: src/server/scripts/Spells/spell_item.cpp [hunk 1]
```diff
@@ -4940,6 +4940,61 @@ class spell_item_variable_pulse_lightning_capacitor : public AuraScript
...
+    {-6552.295410f, -3493.440674f, 346.607330f},
+    {-6596.803711f, -4662.762695f, 8.942942f},
+    {-10212.599609f, 283.842010f, 2.599370f},
+    {-13942.865234f, 18.312716f, 85.728546f},
+    {-14187.959961f, 701.505981f, 42.390049f},
+    {-7096.048828f, -1202.128784f,354.192566f},
+    {-11887.547852f, -2309.466309f, 173.327576f},
+    {-11928.691406f, -3563.387451f, 218.569733f},
+    {7768.569824f, -2429.679932f, 487.309998f},//map 1
+    {5975.012207f, -1441.295410, 444.294556},
```
Comment (acidmanifesto) [inline src/server/scripts/Spells/spell_item.cpp:4962]: Double to Float. 
{5975.012207f, -1441.295410, 444.294556}
needs to read
{5975.012207f, -1441.295410f, 444.294556f}

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: src/server/scripts/Spells/spell_item.cpp (hunk 1 above)
Comment (cTrinity) [reply src/server/scripts/Spells/spell_item.cpp:4962]: I'm careless

**Comment by cTrinity** [comment]
> where did you take the teleported coordinates from?

https://wowpedia.fandom.com/wiki/The_Last_Relic_of_Argus

File: src/server/scripts/Spells/spell_item.cpp (hunk 1 above)
Comment (acidmanifesto) [reply src/server/scripts/Spells/spell_item.cpp]: i have done that numerous times with coordinates myself.

**Comment by Ovahlord** [comment]
I mean where did you get the exact coordinates from. Locations like these require sniff data

**Comment by cTrinity** [comment]
> I mean where did you get the exact coordinates from. Locations like these require sniff data

Try it in the game, check the coordinates,not sniff data, 
I don't have a data sniffing tool

**Pull request closed without merging**